src/grt/src/fastroute/src/RSMT.cpp

```diff
@@ -318,7 +316,7 @@ void FastRouteCore::fluteNormal(int netID,
       tmp_ys[i] = ys[i] * ((int) (100 * coeffV));
     }
 
-    fluteTree = stt::flutes(d, tmp_xs, tmp_ys, s, acc);
+    fluteTree = flt::flutes(d, tmp_xs, tmp_ys, s, acc);
```

Comment:
FastRoute has its own implementation of flute that adds de coeffV and the congestion-driven flute. I'll add both functions to stt builder and create a new API that calls these functions.

Reply:
Actually, I've found both implementations of flute inside FastRoute uses FR internal structs and I don't see a straightforward way to move them to set builder.
I can at least update the `flutes` call to use the stt_builder_ object, instead of calling it directly.